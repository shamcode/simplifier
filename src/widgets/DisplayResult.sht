<div id="result">{{result}}</div>