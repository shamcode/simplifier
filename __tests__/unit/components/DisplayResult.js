import DisplayResult from '../../../src/components/DisplayResult.sht';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( DisplayResult, {
        result: 'a && b'
    } );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
