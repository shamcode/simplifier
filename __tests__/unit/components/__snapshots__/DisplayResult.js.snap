// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Options": Object {
    "ID": "component",
    "result": "a && b",
  },
  "html": "<div id=\\"result\\">a &amp;&amp; b</div>",
}
`;
